### DISCUSSION TITLE
Github Copilot uses a lot of CPU resources

### DISCUSSION THREAD

**Discussion by duoc2k** [community/community#8916] [Projects and Issues] [answered]
I turned off all vscode extensions but left only Copilot but vscode still eats up a lot of CPU.

**Comment by Krzysztof-Cieslak** [comment, marked as answer]
This is already tracked and discussed in https://github.com/github/feedback/discussions/8861

**Comment by airtonix** [comment]
still using a billion percent of my cpu and **my GPU** for some reason???

**Comment by azusaaqour** [comment]
Edge computing technology.
